mne/io/meas_info.py

```diff
@@ -1975,26 +2025,35 @@ def anonymize_info(info, daysback=None, keep_his=False):
     proc_hist = info.get('proc_history')
     if proc_hist is not None:
         for record in proc_hist:
-            record['block_id']['secs'] = info['meas_date'][0]
-            record['block_id']['usecs'] = info['meas_date'][1]
+            this_t0 = (record['block_id']['secs'], record['block_id']['usecs'])
+            this_t1 = _add_timedelta_to_meas_date(this_t0, -delta_t)
+            record['block_id']['secs'] = this_t1[0]
+            record['block_id']['usecs'] = this_t1[1]
             record['block_id']['machid'][:] = 0
-            record['date'] = info['meas_date']
+            record['date'] = _add_timedelta_to_meas_date(record['date'],
+                                                         -delta_t)
             record['experimenter'] = default_str
 
     hi = info.get('helium_info')
     if hi is not None:
         if hi.get('orig_file_guid') is not None:
             hi['orig_file_guid'] = default_str
         if hi.get('meas_date') is not None:
-            hi['meas_date'] = [info['meas_date'][0],
-                               info['meas_date'][1]]
+            hi['meas_date'] = _add_timedelta_to_meas_date(hi['meas_date'],
+                                                          -delta_t)
 
     di = info.get('device_info')
     if di is not None:
         for k in ('serial', 'site'):
             if di.get(k) is not None:
                 di[k] = default_str
-
+    try:
+        info._check_consistency()
+    except RuntimeError as e:
```

Comment:
can we avoid try-except clause somehow?

Reply:
Pushed a change that avoids this but seems abit kludgy.

Either is fine with me...